roiextractors/extractors/nwbextractors/nwbextractors.py

```diff
@@ -180,82 +180,65 @@ def __init__(self, file_path):
         self.image_masks = None
         self._roi_locs = None
         self._accepted_list = None
+        self._rejected_list = None
         self._io = NWBHDF5IO(file_path, mode='r+')
-        nwbfile = self._io.read()
-        self.nwbfile = nwbfile
-        _nwbchildren_type = [type(i).__name__ for i in nwbfile.all_children()]
-        _nwbchildren_name = [i.name for i in nwbfile.all_children()]
-        _procssing_module = [_nwbchildren_name[f]
-                             for f, u in enumerate(_nwbchildren_type) if u == 'ProcessingModule']
-        mod = nwbfile.processing[_procssing_module[0]]
-        if len(_procssing_module) > 1:
-            print('multiple processing modules found, picking the first one')
-        elif not mod:
-            raise Exception('no processing module found')
-
-        # Extract image_mask/background:
-        _plane_segmentation_exist = [i for i, e in enumerate(
-            _nwbchildren_type) if e == 'PlaneSegmentation']
-        if not _plane_segmentation_exist:
-            print('could not find a plane segmentation to contain image mask')
-        else:
-            ps = nwbfile.all_children()[_plane_segmentation_exist[0]]
-        # self.image_masks = np.moveaxis(np.array(ps['image_mask'].data), [0, 1, 2], [2, 0, 1])
-        if 'image_mask' in ps.colnames:
-            self.image_masks = DatasetView(ps['image_mask'].data).lazy_transpose([1, 2, 0])
-        if 'RoiCentroid' in ps.colnames:
-            self._roi_locs = ps['RoiCentroid']
-        if 'Accepted' in ps.colnames:
-            self._accepted_list = ps['Accepted'].data[:]
-        # Extract Image dimensions:
-
-        # Extract roi_response:
-        roi_names = [_nwbchildren_name[val]
-                      for val, i in enumerate(_nwbchildren_type) if i == 'RoiResponseSeries']
-        if not roi_names:
-            raise Exception('no ROI response series found')
+        self.nwbfile = self._io.read()
+        self._raw_movie_file_location = None
```

Comment:
this should not be in SegmentationExtractors

Reply:
fixed https://github.com/catalystneuro/roiextractors/pull/46/commits/2ff20ad290673fe57f0b1ba04b416dcc2a30dd42